### PR TITLE
Feat/#tc-23 setup local storage

### LINKED ISSUE #192: Set up local storage

### PR DESCRIPTION
**Pull request by Kwandes**
### This pull request closes:

✔️ Closes: #192
🏁 Jira Issue: [TC-23](https://treecreate.atlassian.net/jira/software/c/projects/TC/boards/1?modal=detail&selectedIssue=TC-23)

### Which service(s) does this issue affect:

- [ ] 🛰️ API
- [x] 🛒 webstore
- [x] 🎓 admin-page

### Types of changes

- [ ] 🐛 Bug fix (a non-breaking change which fixes an issue)
- [x] 🧱 New feature (a non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🤖 CI/CD (a change to the CI/CD pipeline)
- [ ] ⚗️ Refactoring (an update to some already existing code)

### Documentation Updates

- [x] 📂 No need for updates
- [ ] 📁 Requires an update ( _not done within this issue_ )
- [ ] 🗃️ Has been updated ( _done within this issue_ )

### Testing checklist

- [x] 💪 Manual tests
- [x] 🔧 Automatic tests

#### Browser compatibility - _**[**Frontend only**]**_

- [ ] Tested on Safari
- [x] Tested on Chrome
- [ ] Tested on Mozilla Firefox

### What changes have been done within this issue?

Create a new library `local-storage`
Move the local storage service from webstore to the new library
Replace usages of the old service with the library

### How should this be manually tested?

Run the website and perform any action that would result in changes to local storage (for example, accept cookies, log in, save design)

### CODE REVIEW

File: libs/local-storage/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# local-storage
+
+Local storage library contains a service used to manage information saved in the browsers local storage
```
Comment (CristiPV) [inline libs/local-storage/README.md]:
```suggestion
Local storage library contains a service used to manage information saved in the browser's local storage
```

File: workspace.json [hunk 2]
```diff
@@ -296,107 +262,13 @@
...
-            "lintFilePatterns": ["libs/assets/**/*.ts"]
-          }
-        },
-        "test": {
-          "executor": "@nrwl/jest:jest",
-          "outputs": ["coverage/libs/assets"],
-          "options": {
-            "jestConfig": "libs/assets/jest.config.js",
-            "passWithNoTests": true
-          }
```
Comment (CristiPV) [inline workspace.json:399]: Why did lines 309 -> 399 get removed?

**Review by CristiPV — CHANGES_REQUESTED** [review]

File: libs/local-storage/README.md (hunk 1 above)
Comment (Kwandes) [reply libs/local-storage/README.md]: Resolved in 3c057df

File: workspace.json (hunk 2 above)
Comment (Kwandes) [reply workspace.json:399]: Seems like when you formatted the files in your #211 pull request you have formatted the workspace file with your formatter. It seems like we have different formatters (I use prettier), which we need to resolve asap or we will keep having a lot of formatting changes in each pull request.

The newly formatted files have messed up a tiny bit after a merge, and then when my formatter run it removed your changes while keeping what is needed.
I also run a NX command in that commit, and whenever a NX command is run it goes through the files and does some cleanup, so it has made sure the workspace file is correct

tl dr - formatting fuck up, code works and NX took care of it

Comment (Kwandes) [reply workspace.json:399]: I have double-checked, none of the libraries is missing and everything works correctly

**Review by CristiPV — APPROVED** [review]
